js/lib/alert-bar.js

```diff
@@ -0,0 +1,16 @@
+var $ = require('jquery')
```

Comment:
Let's not import jquery just to create a pop up

http://youmightnotneedjquery.com/

Reply:
Jquery wouldn't be my first choice either, but it is already used in ipfs.io site.

Reply:
ya, we’re already using it on two other pages. if someone wants to send a PR to remove it’s use throughout the site, fine, but I don’t see why we would hold up new contributions since it’s not actually adding a dependency.

Reply:
> Jquery wouldn't be my first choice either, but it is already used in ipfs.io site.

Understood. Although I would argue that the best way to default not be the first choice anymore is to move away from it rather than adding it in more places.